torchvision/prototype/models/resnet.py

```diff
@@ -127,6 +137,16 @@ class ResNet152Weights(Weights):
             "acc@5": 94.046,
         },
     )
+    ImageNet1K_RefV2 = WeightEntry(
+        url="https://download.pytorch.org/models/resnet152-089c0848.pth",
+        transforms=partial(ImageNetEval, crop_size=224, resize_size=232),
```

Comment:
the resize_size was not used in the original training?

Reply:
Nope, that's a new optimization only available on the new weights. :-)